docs/modules/ROOT/nav.adoc

```diff
@@ -59,19 +56,27 @@
 ** xref:cluster-performance:near-cache.adoc[]
 ** xref:cluster-performance:imap-bulk-read-operations.adoc[]
 ** xref:cluster-performance:data-affinity.adoc[]
-include::architecture:partial$nav.adoc[]
 * Member/Client Discovery
 ** xref:clusters:discovery-mechanisms.adoc[]
 ** xref:clusters:discovering-by-tcp.adoc[]
 ** xref:clusters:discovering-by-multicast.adoc[]
 ** xref:clusters:discovering-native-clients.adoc[]
 include::kubernetes:partial$nav.adoc[]
 include::data-structures:partial$nav.adoc[]
-* Distributed Compute
+// Distributed Computing
+* xref:computing:distributed-computing.adoc[]
+** xref:data-structures:entry-processor.adoc[]
 ** Executor Services
 include::computing:partial$nav.adoc[]
+// ** UCN (both parts must go together)
 ** xref:clusters:user-code-namespaces.adoc[]
 include::clusters:partial$nav.adoc[]
+// ** Pipelines
+** xref:pipelines:overview.adoc[Data Pipelines]
```

Comment:
If we put this here, it's also in the pipelines partial nav so you get duplicate entries and funny behaviour e.g. if you click About Data Pipelines in the partial nav it jumps you to here in the nav. Suggest we remove this dupe item but should we consider moving Pipelines section up?